apps/www/registry/new-york/ui/navigation-menu.tsx

```diff
@@ -1,10 +1,21 @@
 import * as React from "react"
 import { ChevronDownIcon } from "@radix-ui/react-icons"
 import * as NavigationMenuPrimitive from "@radix-ui/react-navigation-menu"
+import {
+  NavigationMenuListProps,
+  NavigationMenuViewportProps,
+} from "@radix-ui/react-navigation-menu"
 import { cva } from "class-variance-authority"
 
 import { cn } from "@/lib/utils"
 
+interface NavigationViewPortExntededProps extends NavigationMenuViewportProps {
+  orientation?: "horizontal" | "vertical" | undefined
+}
+interface NavigatioListExtendedProps extends NavigationMenuListProps {
```

Comment:
Mistyped interface name. Missing an "n" at the end of "Navigation"

NavigatioListExtendedProps ==> NavigationListExtendedProps